doc/plugins/org.polarsys.capella.ui.doc/html/First steps with Capella/3.8. How to import Capella Project Example.mediawiki

```diff
@@ -1,15 +1,23 @@
-
-= How to import a Capella Project Example? =
-
-# '''File -> New -> Example ...'''
-# Select '''Capella Project Example'''
-# The wizard dialog appears and proposes a list of available examples. An example decorated by a star means that it was already imported into your workspace, you have to click on the '''Rename''' button in order to be able to import it again. 
-
-[[Image:Images/3.8.%20How%20to%20import%20Capella%20Project%20Example_1.png]]
-[[Image:Images/3.8.%20How%20to%20import%20Capella%20Project%20Example_2.png]]
-
-= How to add an existing Capella project as an example? =
-# The sample project must be a zip containing the project folder
-# The sample project shall be placed in sample folder like '''CAPELLA_INSTALLATION_PATH/samples/model.zip''' or located under a sub folder like '''CAPELLA_INSTALLATION_PATH/samples/model/model.zip'''
-# The import sample wizard dialog will propose a list of available examples including the newly added ones.
-[[Image:Images/3.8.%20How%20to%20add%20new%20Capella%20Project%20Example.png]]
+
+= How to import a Capella Project Example? =
+Since Capella 1.3.0 (Melody 4.4.0), sample projects can be imported using File / Import, then Capella Project Example:

```

Comment:
Can you remove "Since Capella 1.3.0 (Melody 4.4.0), " as it talks about old versions. 
globally, Melody keyword shall not appear as it is not opensource